mule-http-connector/src/test/munit/cookies/request-cookies-redirect-test-case.xml

```diff
@@ -26,9 +26,7 @@
         <http:request-connection host="localhost" port="${dynamic.port}"/>
     </http:request-config>
 
-    <!-- TODO (W-17405565): Un-ignore this test for Netty impl -->
-    <munit:test name="cookiesWithMultipleRedirect" description="Cookies are replaced during the redirect mechanism"
-                ignore="#[p('mule.http.service.implementation') == 'NETTY']">
+    <munit:test name="cookiesWithMultipleRedirect" description="Cookies are replaced during the redirect mechanism]" ignore="#[Munit::muleVersionPriorTo('4.9.3')]">
```

Comment:
Only ignore the test for versions older than 4.9.3 when the implementation is NETTY. Otherwise, you're ignoring this test for GRIZZLY implementation too.

Reply:
And do the same for the rest of the tests